src/workerd/server/server.c++

```diff
@@ -2565,6 +2566,83 @@ void Server::abortAllActors() {
   }
 }
 
+kj::Path getPyodideBundlePath(kj::String &version) {
```

Comment:
Maybe `getPyodideBundleFileName`? I guess it's returning a path, but it's a single component path.

Reply:
Also I think the idomatic kj thing is to accept a `StringPtr` rather than a `String&`.
```suggestion
kj::Path getPyodideBundlePath(kj::StringPtr version) {
```